hgctpgvalidation/display/graphFunctionsMulticonfigs.py

```diff
@@ -0,0 +1,482 @@
+#! /usr/bin/env python
```

Comment:
graphFunctionsMulticonfigs.py displays Memory&Time information into a table. The name “Multiconfigurations” is in fact confusing. I used this name because this new feature was introduced at the same time as the multiconfig version of the code. I will rename graphFunctionsMulticonfigs.py tp graphFunctions.py

Reply:
Ok. Does it mean that everything inside the existing `graphFunctions.py` file is not needed anymore?